Subject: [PATCH] src/02_sum_avg.py
---
Index: src/1zdn.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/1zdn.py b/src/1zdn.py
--- a/src/1zdn.py	(revision 32b393aca8298e5882b908625f3ea5bd65453b2f)
+++ b/src/1zdn.py	(date 1758565509744)
@@ -1,3 +1,6 @@
-name=input("Имя: ")
-age=int(input("Возраст: "))
-print(f'Привет, {name}! Через год тебе будет {age+1}.')
\ No newline at end of file
+a=input('a:')
+b=input('b:')
+a=a.replace(',','.',1)
+b=b.replace(',','.',1)
+a=float(a)
+b=float
\ No newline at end of file
